lib/types.ts

```diff
@@ -66,15 +67,17 @@ export namespace ExpectedExports {
    * Every time a package completes an install, this function is called before the main.
    * Can be used to do migration like things.
    */
-  export type init = (options: {
+  export type init<WD> = (options: {
```

Comment:
This is not what we do for expected exports, will remove. Move to the init & uninit in the migrations fn